src/aktualizr_secondary/aktualizr_secondary_test.cc

```diff
@@ -192,16 +272,16 @@ class SecondaryTestNegative : public ::testing::Test,
  *
  * see INSTANTIATE_TEST_SUITE_P for the test instantiations with concrete parameter values
  */
-TEST_P(SecondaryTestNegative, MalformedMetadaJson) {
-  EXPECT_FALSE(_secondary->putMetadata(currentMetadata()));
+// TEST_P(SecondaryTestNegative, MalformedMetadaJson) {
+//  EXPECT_FALSE(secondary_->putMetadata(currentMetadata()));
 
-  EXPECT_CALL(_update_agent, download).Times(0);
-  EXPECT_CALL(_update_agent, install).Times(0);
+//  EXPECT_CALL(_update_agent, download).Times(0);
+//  EXPECT_CALL(_update_agent, install).Times(0);
 
-  EXPECT_FALSE(_secondary->sendFirmware("firmware"));
+//  EXPECT_FALSE(secondary_->sendFirmware("firmware"));
 
-  EXPECT_NE(_secondary->install("target"), data::ResultCode::Numeric::kOk);
-}
+//  EXPECT_NE(secondary_->install("target"), data::ResultCode::Numeric::kOk);
+//}
```

Comment:
More commented out stuff... is this still WIP?

Reply:
no, I am uncommenting it.